### ISSUE TITLE
No softmax layer

### ISSUE THREAD

**Issue by JiyueWang** [weiaicunzai/pytorch-cifar100#26]
Is there no softmax layer for all models? Will this problem influence the test acc?

**Comment by sangenan** [comment]
Do you already know the reason why no softmax layer for all models?

**Comment by JiyueWang** [comment]
Yes, because the softmax layer is included in the cross-entropy loss function in PyTorch.

**Comment by sangenan** [comment]
thanks a lot
